lib/src/common/enums.dart

```diff
@@ -0,0 +1,430 @@
+part of tiled;
+
+enum MapOrientation { orthogonal, isometric, staggered, hexagonal }
+
+extension MapOrientationExtension on MapOrientation {
+  String get name {
+    switch (this) {
+      case MapOrientation.orthogonal:
+        return 'orthogonal';
+      case MapOrientation.isometric:
+        return 'isometric';
+      case MapOrientation.staggered:
+        return 'staggered';
+      case MapOrientation.hexagonal:
+        return 'hexagonal';
+    }
+  }
+}
+
+extension MapOrientationParser on Parser {
+  MapOrientation? getMapOrientationOrNull(
```

Comment:
The `get` part can de dropped for almost all getters in this file

Reply:
oh boy, you mean changing the entire Parser class? haha
I think in this case I wanted to be more explicit with a verb because it wasn't just a getter, it's actually parsing the xml/json
also would it work to just name it `int` for example? since that is a type name?

Reply:
True, since it is all about types here maybe it is good to have `get` in-front.